src/app/compressed-air-assessment/results-panel/help-panel/help-panel.component.ts

```diff
@@ -11,14 +11,22 @@ export class HelpPanelComponent implements OnInit {
 
   selectedTab: string;
   selectedTabSub: Subscription;
+  assessmentTab: string;
+  assessmentTabSub: Subscription;
+
+  mainTab: string;
   constructor(private compressedAirService: CompressedAirAssessmentService) { }
 
   ngOnInit(): void {
-    let mainTab: string = this.compressedAirService.mainTab.getValue();
-    if (mainTab == 'system-setup') {
+    this.mainTab = this.compressedAirService.mainTab.getValue();
+    if (this.mainTab == 'system-setup') {
       this.selectedTabSub = this.compressedAirService.setupTab.subscribe(val => {
         this.selectedTab = val;
       });
+    } else if (this.mainTab == 'assessment') {
+      this.assessmentTabSub = this.compressedAirService.assessmentTab.subscribe(val => {
```

Comment:
Need to also check for and unsubscribe from assessmentTabSub in the ngOnDestroy method